service/service.go

```diff
@@ -56,10 +56,17 @@ var getObservationStore = func(ctx context.Context) (observationStore *graph.DB,
 	return observationStore, errorConsumer, nil
 }
 
+var pConfig *kafka.ProducerConfig
+
+var cfg *config.Config
+
 // getProducer returns a kafka producer
-var getProducer = func(ctx context.Context, kafkaBrokers []string, topic string, envMax int) (kafkaProducer kafka.IProducer, err error) {
+var getProducer = func(ctx context.Context, kafkaBrokers []string, topic string) (kafkaProducer kafka.IProducer, err error) {
+	pConfig := &kafka.ProducerConfig{
+		KafkaVersion: &cfg.KafkaVersion,
```

Comment:
We should provide MaxMessageBytes to producer config, as it is provided by the general config:
```
		MaxMessageBytes: &cfg.KafkaMaxBytes,
```